docs/website/docs/dlt-ecosystem/verified-sources/sql_database.md

```diff
@@ -118,9 +118,20 @@ Here, we use the `mysql` and `pymysql` dialects to set up an SSL connection to a
    ```toml
    sources.sql_database.credentials="mssql://user:pw@my_host/my_database?trusted_connection=yes"
    ```
-   
+
    >Note: The (user:pw) may be included but will be ignored by the server if `trusted_connection=yes` is set.
 
+1. To pass credentials with ODBC driver, you can modify your connection string as:
```

Comment:
@dat-a-man what this adds on top of the examples above? is this alternative to the windows authentication?
maybe we could refer to this documentation? https://dlthub.com/docs/dlt-ecosystem/destinations/mssql#setup-guide
I compiled the most common problems from our users (mostly SSL, certificates)